recipes/pyomo/meta.yaml

```diff
@@ -0,0 +1,169 @@
+{% set version = "4.3.11377" %}
+
+package:
+  name: pyomo
+  version: {{ version }}
+
+source:
+  fn: Pyomo-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/12/8e/edae55601011b4356f26e3d3346e0032a67a89cd2621ff893e0a623d68ab/Pyomo-4.3.11377.tar.gz
+  #url: https://pypi.python.org/packages/source/P/Pyomo/Pyomo-{{ version }}.tar.gz
```

Comment:
So just remove the commented URL. I am not sure why some URLs are re-direct and others are not. This change in PyPI is really annoying.